### ISSUE TITLE
manageDocker failing for task role based codeontap deployment

### ISSUE THREAD

**Issue by roleyfoley** [codeontap/gen3-automation#115]
When running the manageDocker.sh script from a container based codeontap jenkins agent. The local registry management fails with the error: 

```
16:25:12 [EnvInject] - Variables injected successfully.
16:25:12 [1-Pull-Image] $ /bin/bash /tmp/jenkins5010240702152116372.sh
16:25:12 /opt/codeontap/automation/jenkins/aws/manageDocker.sh: line 262: : bad substitution
16:25:12 
16:25:12 (Fatal) Can't log in to 340127619209.dkr.ecr.ap-southeast-2.amazonaws.com
16:25:12 Build step 'Execute shell' marked build as failure
16:25:16 Finished: FAILURE
```

Confirmed that the configuration is ok but it looks like the issue might be with using a taskRole for aws permissions.

**Comment by ml019** [comment]
@roleyfoley think this is fixed now? Please close if so

**Comment by roleyfoley** [comment]
fixed by #113
